pilot/pkg/security/authz/policy/v1alpha1/v1alpha1_test.go

```diff
@@ -47,21 +47,21 @@ func TestV1alpha1Generator_Generate(t *testing.T) {
 			name: "no role in namespace a",
 			policies: []*model.Config{
 				policy.SimpleRole("role", namespaceB, serviceFoo),
-				policy.SimpleBinding("binding", namespaceB, "role"),
+				policy.SimpleBinding("binding", namespaceB, "role", policy.SimplePrincipal("binding")),
```

Comment:
It's really a pain looking at these extra call to `policy.SimplePrincipal`, especially it's used in 90% cases, could you refactor to avoid these duplicate calling?